class Exceptions::LogicException < Exception
  def initialize(message)
    super(message)
  end
end